libs/Microsoft.MixedReality.WebRTC.Unity/Packages/manifest.json

```diff
@@ -1,8 +1,20 @@
 {
   "dependencies": {
-    "com.unity.package-manager-ui": "2.0.8",
-    "com.unity.textmeshpro": "1.4.1",
+    "com.unity.2d.sprite": "1.0.0",
+    "com.unity.2d.tilemap": "1.0.0",
+    "com.unity.ext.nunit": "1.0.0",
+    "com.unity.ide.rider": "1.1.0",
+    "com.unity.ide.vscode": "1.1.2",
+    "com.unity.multiplayer-hlapi": "1.0.2",
+    "com.unity.package-manager-ui": "2.2.0",
+    "com.unity.test-framework": "1.0.13",
+    "com.unity.textmeshpro": "2.0.1",
+    "com.unity.timeline": "1.1.0",
+    "com.unity.ugui": "1.0.0",
```

Comment:
Probably don't need most of the new ones here (aside from `nunit`)

Reply:
These must have been auto-added by Unity. Likely due to my use of Unity 2019.2.12f1. I will revert this, and install 2018 to avoid such side effects.

Reply:
They're auto-added, can't remember under which circumstances, but can be safely removed manually from the Package Manager UI.